src/mypreferencesquickconnect.html

```diff
@@ -0,0 +1,11 @@
+<div id="quickConnectPreferencesPage" data-role="page" class="page libraryPage userPreferencesPage noSecondaryNavPage" data-title="${HeaderHome}" data-backbutton="true">
+    <button is="emby-button" id="btnQuickConnectActivate" type="button" class="raised button-submit block">
+        <span>Activate</span>
+    </button>
+
+    <div class="quickConnectSettingsContainer padded-left padded-right padded-bottom-page">
+        <div id="quickConnectIncoming" class="paperList">
+            <div>Failed to load incoming requests</div>
```

Comment:
These should probably be translated since they're presented to the user. You can add a new translated string by adding it to `en-us.json` and then use it like above (`${HeaderHome}`)